clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/LoggingClientBuilder.java

```diff
@@ -74,14 +79,16 @@ private <T> T getProcessor(Class<T> clazz) {
 
   /**
    * Provides a {@link LoggingClientBuilder} with default {@link LogProcessor}s; {@link
-   * ValidationProcessor}, {@link RuntimeInfoProcessor}, and {@link RemoteProcessor}. }
+   * ValidationProcessor}, {@link RuntimeInfoProcessor}, {@link RemoteProcessor}, and when required,
+   * {@link JustificationProcessor}. }
    */
   public LoggingClientBuilder withDefaultProcessors() {
     return withValidationProcessor()
         .withFilteringProcessor()
         .withRuntimeInfoProcessor()
         .withDefaultLogBackends()
-        .withLabelProcessor();
+        .withLabelProcessor()
+        .withJustificationProcessor();
```

Comment:
Nit: I would rename this to something like `.withJustificationProcessorIfRequired()` in order to convey the fact that if the configuration doesn't require it, it won't be added to mutators even if this is called.

Reply:
Ok, will rename it in the following CR for https://github.com/abcxyz/lumberjack/issues/280 to `.withJustificationProcessorIfEnabled` where the env var is also changed (from REQUIRED to ENBLED).

Reply:
SG, thanks.